README.md

```diff
@@ -104,27 +96,18 @@ To fix all Linter error automatically
 - GitHub: [@chelobotix](https://github.com/chelobotix)
 - Linkedin: [Marcelo Alarcon](https://www.linkedin.com/in/marceloalarconb/)
 
-👤 **Author**
-
-: Farida Gueleu
-
-ca Cameroon 💓
-
-- GitHub: [@SFarida](https://github.com/SFarida)
-- Linkedin: [Farida Gueleu](https://www.linkedin.com/in/farida-gueleu/)
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- [ ] **UI**
+- [ ] **More Classes**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
-Feel free to check the [issues page](https://github.com/SFarida/enumerable/issues).
+Feel free to check the [issues page](https://github.com/chelobotix/school-library/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] I think it would be better if you fix the broken `[MIT](<a href="./licence">License</a>)` and `<a href="./licence">License</a>` links in your README file to make it more professional as it shows a **404** error page once clicked and use only one link. And to fix that, you can only create a new file name `LICENSE` in your root directory and get its content from this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)** or rename this `[MIT](./LICENSE)` to `[MIT](./MIT.md)`, then the created file in your directory name it `MIT.md` and use the same comment in this **[template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)**

- [ ] I also think it would be better if you remove the below text after your `License` link that said: `Original design idea by <a href="https://www.behance.net/sakwadesignstudio">Nelson Sakwa on Behance</a>.` as you don't use any design from  `Behance` and to make all inforrmation of your documentation to match the project.